packages/wdio-junit-reporter/src/index.js

```diff
@@ -29,7 +29,7 @@ class JunitReporter extends WDIOReporter {
             ? `${runner.sanitizedCapabilities}-${this.options.packageName}`
             : runner.sanitizedCapabilities
 
-        let index = 0
+        const index = 0
```

Comment:
I don't understand, so index will be always 0? Can we add a comment for it as to why?